src/reducers/clusterReducer.js

```diff
@@ -129,7 +129,7 @@ const clusterReducer = produce((draft, action) => {
       return;
 
     case types.CLUSTER_DELETE_SUCCESS:
-      delete draft.items[action.clusterId];
+      draft.items[action.clusterId].delete_date = Date.now();
```

Comment:
`Date.now()` makes this reducer impure (heresy!)  😛 

As we are not writing unit tests for reducers I am ok with this, but it is a no-no in reduxland: https://github.com/reduxjs/redux/issues/1639#issuecomment-212107524

Reply:
I could pass it in from the action. Let's try to do the right thing :) 

There's still those last updated things with Date.now, but I don't wanna make the problem worse.